csm/conf/configure.py

```diff
@@ -324,16 +325,14 @@ def _configure_csm_ldap_schema(self):
         ldap_url = Setup._get_ldap_url()
```

Comment:
Please check if 'ldap_url' is required ?

Reply:
Modifications done